JS Tasks/DataTypes/Transportation on vacation.js

```diff
@@ -0,0 +1,11 @@
+function rentalCarCost(d) {
+    // Your solution here
+    var total=0;
+  if(d==7||d>7)

```

Comment:
you can use ternary operator or when we use return inside if then else is not required